tutorials-and-examples/nvidia-nim/blueprints/digitalhuman/README.md

```diff
@@ -0,0 +1,140 @@
+# Digital Human for Customer Service on GKE
+
+Deploying the digital human blueprint based on few NIMs on GKE.
+
+## Table of Contents
+
+- [Digital Human for Customer Service on GKE](#digital-human-for-customer-service-on-gke)
+  - [Table of Contents](#table-of-contents)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Tear down](#tear-down)
+
+## Prerequisites
+
+- **GCloud SDK:** Ensure you have the Google Cloud SDK installed and configured.
+- **Project:**  A Google Cloud project with billing enabled.
+- **NGC API Key:** An API key from NVIDIA NGC. Please read the prerequisites to access this key [here](https://github.com/NVIDIA-AI-Blueprints/digital-human/blob/main/README.md#prerequisites)
+- **kubectl:**  kubectl command-line tool installed and configured.
+- **NVIDIA GPUs:** One of the below GPUs should work
+  - [NVIDIA L4 GPU (8)](https://cloud.google.com/compute/docs/gpus#l4-gpus)
+  - [NVIDIA A100 80GB (1) GPU](https://cloud.google.com/compute/docs/gpus#a100-gpus)
+  - [NVIDIA H100 80GB (1) GPU or higher](https://cloud.google.com/compute/docs/gpus#a3-series)
+
+## Setup
+
+1. **Environment setup**: You'll set up several environment variables to make the following steps easier and more flexible. These variables store important information like cluster names, machine types, and API keys. You need to update the variable values to match your needs and context.
+
+    ```bash
+    gcloud config set project "<GCP Project ID>"
+
+    export CLUSTER_NAME="gke-nimbp-dighuman"
+    export NP_NAME="gke-nimbp-dighuman-gpunp"
+
+    export ZONE="us-west4-a"            # e.g., us-west4-a
+    export NP_CPU_MACHTYPE="e2-standard-2" # e.g., e2-standard-2
+    export NP_GPU_MACHTYPE="g2-standard-96" # e.g., a2-ultragpu-1g
+
+    export ACCELERATOR_TYPE="nvidia-l4"     # e.g., nvidia-a100-80gb
+    export ACCELERATOR_COUNT="8"            # Or higher, as needed
+    export NODE_POOL_NODES=1                # Or higher, as needed
+
+    export NGC_API_KEY="<NGC API Key>"
+
+    ```
+
+2. **GKE Cluster creation**:
+
+    ```bash
+
+    gcloud container clusters create "${CLUSTER_NAME}" \
+      --num-nodes="1" \
+      --location="${ZONE}" \
+      --machine-type="${NP_CPU_MACHTYPE}" \
+      --gateway-api=standard \
```

Comment:
I can't tell from the docs, but it doesn't look like this is enabled by default - can you confirm?